### PR TITLE
Add links/reference to new multimodal instructblip-pipeline in multimodal readme

### PR DESCRIPTION
**Pull request by kjerk**
Simply adding links/references in the multimodal readme.

tl;dr is: New pipeline for Vicuna-7b and 13b interop for multimodal interrogation, including almost any related 7b/13b networks like `nous-hermes` or `airoboros`, though the vanilla 1.1 networks are the 'canonically' correct ones, the context transfers well enough to get a good user experience out of most related or downhill networks.

This is a pipeline for one of the SOTA (at least for the next few weeks 😭) image captioning networks called [InstructBLIP](https://github.com/salesforce/LAVIS/tree/main/projects/instructblip). It was extremely taxing to run with vanilla transformers (7b maxing out a 4090, whyyyyy) so wanted it running on quantized models ASAP because the quality can be very high.

Thanks to this repo, the multimodal pipeline abstraction, and all the work on bitsandbytes/AutoGPTQ, now 7b can run on 8gb and 13b can run on 12gb.

Pipeline repo is here, with a fully fleshed out readme: https://github.com/kjerk/instructblip-pipeline/tree/main#-examples

---

Edit: Examples
example 1

example 2

### CODE REVIEW

**Comment by oobabooga** [comment]
Sorry for taking so long to review, this is super relevant. Thank you!!
